projects/components/src/form/form-input/form-input.component.scss

```diff
@@ -17,4 +17,8 @@
 
 clr-signpost p:first-child {
     margin-top: 0;
+
+    // Sequences of whitespace will collapse into a single whitespace.
+    // Text will wrap when necessary, and on line breaks.
+    white-space: pre-line;
```

Comment:
This is not the same as what we use in https://github.com/vmware/vmware-cloud-director-ui-components/blob/a11y/projects/components/src/form/number-with-unit-input/number-with-unit-form-input.component.scss#L1 Can we make them be the same? Should we use a separate CSS file that is included by both?


Shouldn't we fix this on form-checkbox which also has a signpost? Please see https://github.com/vmware/vmware-cloud-director-ui-components/blob/a11y/projects/components/src/form/form-checkbox/form-checkbox.component.html#L21